docs/user-guide/redis-deployment.yaml

```diff
@@ -1,8 +1,11 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: redis
 spec:
+  selector:
+    matchLabels:
+      app: redis
```

Comment:
and `tier: backend` (same comment for the other redis deployments)

Reply:
Done. Thank you for your reply.